hudi-flink/src/main/java/org/apache/hudi/source/JsonStringToHoodieRecordMapFunction.java

```diff
@@ -65,10 +67,12 @@ public HoodieRecord map(String value) throws Exception {
   private void init() {
     this.props = StreamerUtil.getProps(cfg);
     avroConvertor = new AvroConvertor(new FilebasedSchemaProvider(props).getSourceSchema());
+    String keyGeneratorClass = props.getString("hoodie.datasource.write.keygenerator.class", SimpleAvroKeyGenerator.class.getName());
     try {
       keyGenerator = StreamerUtil.createKeyGenerator(props);
+      LOG.info("KeyGenerator {} initialized successfully", keyGeneratorClass);
```

Comment:
Should it be `info` level? What about `debug` level?

Reply:
> Should it be `info` level? What about `debug` level?

how about remove this log ?

Reply:
+1